packages/edit-site/src/components/layout/router.js

```diff
@@ -75,6 +75,31 @@ function useRedirectOldPaths() {
 	}, [ history, params ] );
 }
 
+const GLOBAL_STYLES_PATH_PREFIX = '/wp_global_styles';
+
+function GlobalStylesUIWrapper() {
```

Comment:
For me we should move this component/folder outside of the "router.js" file, the router.js file should be just about "route registration" and not have a lot of logic/components.

Reply:
Good point I will do the change.